### ISSUE TITLE
ERROR Metadata file 'System.Core' could not be found

### ISSUE THREAD

**Issue by thomaslevesque** [mono/t4#3]
I have a pretty simple T4 file like this:

```
<#@ template debug="false" hostspecific="true" language="C#" #>
<#@ assembly name="System.Core" #>
<#@ import namespace="System.IO" #>
<#@ import namespace="System.Linq" #>
<#@ output extension=".cs" #>
<#
    string versionFile = Host.ResolvePath("version.txt");
    string version = File.ReadLines(versionFile).First();
#>
using System.Reflection;

[assembly: AssemblyVersion("<#= version #>")]
[assembly: AssemblyFileVersion("<#= version #>")]
```

When I try to transform it with Mono.TextTransform, I get the following error:

```
> C:\My\Solution [master ≡]> . "C:\Users\thomas\.nuget\packages\Mono.TextTransform\1.0.0\tools\TextTransform.exe" Properties\VersionInfo.tt -out Properties\VersionInfo.cs
Processing 'Properties\VersionInfo.tt' failed.
(0,0): ERROR Metadata file 'System.Core' could not be found
```

**Comment by SamuelDebruyn** [comment]
I have the same issue...

**Comment by mhutch** [comment]
This should be fixed in 2.0 as it improves framework assembly resolution behavior.
